create_grids/incident/sps/install_maraston11_pickles.py

```diff
@@ -0,0 +1,154 @@
+    tar.close()
+    os.remove(filename)
+
+
+
+def make_grid(model, imf, extension):
+    """Main function to convert Maraston 2011 and
+    produce grids used by synthesizer"""
+
+    #synthesizer_model_name = get_model_filename(model)
+    #print(synthesizer_model_name)
+
+    # Define output
+    fname = f"{synthesizer_data_dir}/input_files/{model_name}/{model_name}{extension}_{imf}.hdf5" #ideally find a way to create the grids/model_name folder if it doesn't already exist
+
+    metallicities = np.array([0.02])  # array of available metallicities
+
+    log10metallicities = np.log10(metallicities)
+
+    metallicity_code = {0.02: "002"}  # codes for converting metallicty
+
+    fn = f"{input_dir}/ssp_M11_Pickles{extension}.{imf_code[imf]}z{metallicity_code[metallicities[0]]}"
+
+    ages_, _, lam_, flam_ = np.loadtxt(fn).T # flam is in (ergs /s /AA /Msun)
+
+    ages_Gyr = np.sort(np.array(list(set(ages_)))) # Gyr 
+    ages = ages_Gyr * 1e9  # yr
+    log10ages = np.log10(ages) 
+
+    lam = lam_[ages_ == ages_[0]]
+
+    flam_ = flam_ * (10**(-8))**2 / c #to convert from (ergs /s /AA /Msun) to (ergs /s /Hz /Msun)
+
+    spec = np.zeros((len(ages), len(metallicities), len(lam)))
+
+    for iZ, metallicity in enumerate(metallicities):
+        for ia, age_Gyr in enumerate(ages_Gyr):
+            print(iZ, ia, fn)
+            ages_, _, lam_, flam_ = np.loadtxt(fn).T
+
+            flam = flam_[ages_ == age_Gyr]
+            fnu = flam_to_fnu(lam, flam)
+            spec[ia, iZ] = fnu
+
+    #write out spectra 
+    write_data_h5py(fname, "spectra/wavelength", data=lam, overwrite=True)
+    write_attribute(
+        fname, "spectra/wavelength", "Description", "Wavelength of the spectra grid"
+    )
+    write_attribute(fname, "spectra/wavelength", "Units", "AA")
+
+    write_data_h5py(fname, "spectra/incident", data=spec, overwrite=True)
+    write_attribute(
+        fname,
+        "spectra/incident",
+        "Description",
+        "Three-dimensional spectra grid, [age, metallicity, wavelength]",
+    )
+    write_attribute(fname, "spectra/incident", "Units", "erg s^-1 Hz^-1")
+
+
+    #write out axes
+    write_attribute(fname, '/', 'axes', ('log10age','metallicity'))
+
+    write_data_h5py(fname, "axes/log10age", data=log10ages, overwrite=True)
+    write_attribute(
+        fname, "axes/log10age", "Description", "Stellar population ages in log10 years"
+    )
+    write_attribute(fname, "axes/log10age", "Units", "log10(yr)")
+
+    write_data_h5py(fname, "axes/metallicity", data=metallicities, overwrite=True)
+    write_attribute(fname, "axes/metallicity", "Description", "raw abundances")
+    write_attribute(fname, "axes/metallicity", "Units", "dimensionless [Z]")
+
+    return fname
+
+
+
+# Lets include a way to call this script not via an entry point
+if __name__ == "__main__":
+    
+    synthesizer_data_dir = '/home/sophienewman/code/synthesizer/src/synthesizer/data/synthesizer_data' #find smart way to do this
```

Comment:
Rather than hard coding the directory here, could you read in the directory from command line arguments, as shown here? 

https://github.com/flaresimulations/synthesizer-grids/blob/10ac5d1ba482879e1a6b593130526257e75d110d/create_grids/incident/sps/install_bc03.py#L258